src/platform_impl/web/web_sys/canvas/pointer_handler.rs

```diff
@@ -103,11 +111,91 @@ impl PointerHandler {
         ));
     }
 
+    pub fn on_pointer_move<F>(&mut self, canvas_common: &super::Common, mut handler: F)
+    where
+        F: 'static + FnMut(i32, PhysicalPosition<f64>),
+    {
+        self.on_pointer_move = Some(canvas_common.add_event(
+            "pointermove",
+            move |event: PointerEvent| {
+                handler(
+                    event.pointer_id(),
+                    PhysicalPosition {
+                        x: event.offset_x() as f64,
+                        y: event.offset_y() as f64,
+                    },
+                );
+            },
+        ));
+    }
+
+    pub fn on_pointer_down<F>(&mut self, canvas_common: &super::Common, mut handler: F)
+    where
+        F: 'static + FnMut(i32, PhysicalPosition<f64>),
+    {
+        let canvas = canvas_common.raw.clone();
+        self.on_pointer_down = Some(canvas_common.add_event(
+            "pointerdown",
+            move |event: PointerEvent| {
+                handler(
+                    event.pointer_id(),
+                    PhysicalPosition {
+                        x: event.offset_x() as f64,
+                        y: event.offset_y() as f64,
+                    },
+                );
+                canvas
+                    .set_pointer_capture(event.pointer_id())
+                    .expect("Failed to set pointer capture");
```

Comment:
I'm not sure if panicking is the correct response to failure here. I'd honestly be surprised myself if my (hypothetical) application crashed because the browser didn't let Winit use `Element.setPointerCapture` for some reason.

Also: I don't quite understand why using `Element.setPointerCapture` is necessary here in the first place.

Reply:
I've copied this from the `on_mouse_press` event in the same file. These events are very similar, so I thought this was required. I'll remove it if it's not necessary.